index.js

```diff
@@ -144,6 +144,21 @@ bot.on("message", msg => {
   }
 });
 
+bot.on("message", msg => {
+  //welcome greeting
+  if (msg.new_chat_members) {
+    var out = "Welcome ";
```

Comment:
Use `let` instead of `var`. I will also be making an issue to replace all var with let. I got to learn it the hard way. 😅

Reply:
Rest, LGTM. Thanks for contributing 🙂 💯